src/App.vue

```diff
@@ -1,13 +1,17 @@
 <script setup lang="ts">
-import HelloWorld from '@/components/HelloWorld.vue'
-import TaxCalculatorVue from '@/components/common/TaxCalculator/TaxCalculator.vue'
+import { RouterView, useRoute } from 'vue-router'
+import NavBar from '@/components/NavBar/NavBar.vue'
+import { computed } from 'vue'
+import { Routes } from './router/common'
+
+const route = useRoute()
+
+const showNavBar = computed(() => (route.path === Routes.REGISTER_PAGE ? false : true))
```

Comment:
```suggestion
const showNavBar = computed(() => (route.path !== Routes.REGISTER_PAGE))
```

nit: У нас уже при сравнении происходят преобразования в булиан и не нужно прям явно возвращать значения))
Так же если у нас может быть больше роутов, то лучше создать переменную роутыБезНавбара или любое другое название и проверять что роут входит в список массива или нет, так будет легче потом доабвлять дополнительные условия.

Reply:
Логично) Исправлю

Reply:
Junior moment